akka-http-core/src/main/java/akka/http/javadsl/model/HttpMessage.java

```diff
@@ -55,6 +58,11 @@
      */
     ResponseEntity entity();
 
+    /**
+     *  Drains entity stream of this message
+     */
+    CompletionStage<Done> ignoreDataBytesJ(Materializer materializer);
```

Comment:
It's not as easy to discover, also very easy to make a typo and get the wrong thing back without noticing.
I do think we proper names here.